src/Log.ts

```diff
@@ -1,18 +1,12 @@
-import { IVbrickEmbedConfig } from './IVbrickApi';
-
-let enableLogging = false;
+import { VbrickSDKConfig } from "./VbrickSDK";
 
 export interface ILogger {
 	log(message?: any, ...optionalParams: any[]): void;
 	error(message?: any, ...optionalParams: any[]): void;
 }
 
-export function init(cfg: IVbrickEmbedConfig): void {
-	enableLogging = !!cfg.log;
-}
-
-export function getLogger(): ILogger {
-	return enableLogging ? console : {
+export function getLogger(cfg: VbrickSDKConfig): ILogger {
```

Comment:
I like this. Can we also make it possible to set `cfg.log` to an `ILogger` as well? like "if cfg.log is an object then return that, otherwise if true return console, otherwise noop"

Reply:
Id rather not put it on the cfg. 
the logger is not completely implemented, so I havent found a good place for it yet.